08-transport.Rmd

```diff
@@ -12,7 +12,9 @@ library(tidyverse)
 
 ## Introduction
 
+<!-- I would tone down, why should geographic space be more tangible in transport than in remote sensing, location analysis, ecological modeling, etc.? --> 
 In no other sector is geographic space more tangible than transport.
+<!-- I am not 100% sure but I think the first law of geography was the basis for spatial dependence, spatial autocorrelation, and therefore the foundation for spatial interpolation. Not sure how 'moving' relates to this but this just might be because I am not a transportation guy. -->
```

Comment:
I am not 100% sure but I think the first law of geography was the basis for spatial dependence, spatial autocorrelation, and therefore the foundation for spatial interpolation. Not sure how 'moving' relates to this but this just might be because I am not a transportation guy.

Reply:
It's a term that links all those things together. If you've an alternative suggestion let me know but good to get that in there I think, linking back to the geocomputation lit (before it was called geocomputation when Tobler coined the '1st law').

Reply:
👍